packages/web-app-files/tests/unit/components/SideBar/SideBar.spec.js

```diff
@@ -0,0 +1,295 @@
+/* eslint-disable jest/no-conditional-expect */
+import Vuex from 'vuex'
+import DesignSystem from 'owncloud-design-system'
+import stubs from '@/tests/unit/stubs/index.js'
+import { createLocalVue, mount } from '@vue/test-utils'
+import SideBar from '@files/src/components/SideBar/SideBar.vue'
+
+import GetTextPlugin from 'vue-gettext'
+
+const localVue = createLocalVue()
+localVue.use(DesignSystem)
+localVue.use(Vuex)
+localVue.use(GetTextPlugin, {
+  translations: 'does-not-matter.json',
+  silent: true
+})
+
+const filesPersonalRoute = { name: 'files-personal' }
+
+const getSideBarComponent = function(name) {
```

Comment:
What do you think about making a stub component from this? Like we already have it for OcPageSize and OcSwitch? If you want to keep it as a helper, please move it to the bottom of the file.